File: cw/cpp/is_he_gonna_survive.cpp
// https://www.codewars.com/kata/59ca8246d751df55cc00014c/train/cpp


#include <iostream>

bool hero(int bullets, int dragons) {
    while(dragons != 0) {
        bullets -= 2;
        dragons -= 1;
    }

    if(bullets >= dragons) {
        return true;
    } else {
        return false;
    }
}

int main() {
    int bullets, dragons;

    std::cin >> bullets >> dragons;
    std::cout << hero(bullets, dragons) << "\n";

    return 0;
}

File: cw/python/is_he_gonna_survive.py
# https://www.codewars.com/kata/59ca8246d751df55cc00014c/train/python


def hero(bullets, dragons):
    while(dragons != 0):
        bullets -= 2
        dragons -= 1

    return True if (bullets >= dragons) else False


if __name__ == "__main__":
    bullets = int(input())
    dragons = int(input())